src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -324,18 +421,66 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
         msgs_data = self.gen_messages(orders_data.get_list_of_orders())
         self.respond_to_invalid_orders(invalid_proposal_orders, msgs_data)
         self.respond_to_alliance_messages(msgs_data)
-        msg_allies = ','.join([pow for pow in self.stance.stance[self.power_name] if (pow != self.power_name and self.stance.stance[self.power_name][pow] > 0)])
```

Comment:
add format off comments here

Reply:
Done, I also made the lists of allies, foes and neutrals separate vars for readability.